### DISCUSSION TITLE
Says compatibility mode but I have all the requirements or better.

### DISCUSSION THREAD

**Discussion by aaschmidy** [AppGround-io/bluetooth-keyboard-and-mouse-support#283] [Bugs & Problems]
I've tried connecting in multiple different ways to my work computer, but there's no situation where I can get a connection on the paid version. the moment I switched, the app was unusable. I removed all traces of pairing on both devices multiple times, turned on every possible setting I could think of that allowed sharing, tried to pair using the phone first, using the pc to pair, and using the app to create a connection. Nothing worked. 

The weird thing is that it worked fine as a mouse before I paid for it.... then nothing. I'm running an LG velvet with android 12, and Bluetooth 5.1. Receiving device is a Lenovo think pad. both devices pair fine by themselves, but the app won't work.

**Comment by AdamKane41** [comment]
Silly question: Do you have curently BOTH versions (Blue and Red) of the app installed on your Android device?

**Comment by aaschmidy** [comment]
My good man, when my doctor asks me a question to help fux a problem, I
never stop to wonder whether it's silly. Similarly you should ask whatever
question you need to right down to the ones that treat me like a
incompetent moron.

Regardless, I think some voluntary clarification of my situation is
required.

I used the in app purchase to upgrade the blue version. That said, after I
requested a refund from Google, I retested the functionality of the blue
app last night and I was only able to get the test portion to work: "is the
mouse moving?"

Even then, it successfully paired only once out of (probably) four tries
with deletion of the app, unpairing, and reinstall.

The test function worked the very first time and then that once during the
multiple reinstalls, but once engaged past that point there is no
functionality.

Andy.
